docs/DeveloperGuide.adoc

```diff
@@ -297,86 +299,263 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |user |add a new problem |keep track of the problems for future usage
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |user |delete a problem |remove entries that I no longer need
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |user |find a problem by keyword |locate details of problems without having to go through the entire list
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* *` |user |do advanced search on problems |locate details of problems without having to go through the entire list
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
-|=======================================================================
+|`* *` |user |do fuzzy search on problems |locate details of problems without having to go through the entire list
+
+|`* * *` |user |create custom tags |categorize problems via tags
+
+|`* * *` |user |add tags to problems |categorize problems via tags
+
+|`* *` |user |sort problems according to difficulty |locate problems easily
+
+|`* *` |user |add remarks to problems |have reference in the future
+
+|`* * *` |user |create plans containing problems |better prepare for interview
+
+|`* * *` |user |add problems to a plan |better prepare for interview
+
+|`* * *` |user |mark problems as complete/incomplete within plans |better prepare for interview
 
-_{More to be added}_
+|`* * *` |user |import database from <<json,JSON>> files |easily transfer data from one computer to another
+
+|`* * *` |user |export data into JSON format |easily transfer data from one computer to another
+
+|`* *` |advanced user |export data into CSV format |do some manipulation/processing on the data
+
```

Comment:
Maybe you can add more user stories with regards to some of the features such as edit plan, find plan, done plan.